javaparser-core-testing/src/test/java/com/github/javaparser/modules/ModuleDeclarationTest.java

```diff
@@ -180,4 +181,39 @@ public void testCsmPrinting() {
                         "}" + EOL, ConcreteSyntaxModel.genericPrettyPrint(cu));
 
     }
+
+    @Test
+    public void fluentInterface() {
+        ModuleDeclaration moduleDeclaration = new CompilationUnit()
+                .addModule("com.laamella.base")
+                .addSingleMemberAnnotation(SuppressWarnings.class, "\"module\"")
+                .addDirective("requires transitive java.desktop;")
+                .addDirective("exports com.laamella.base.entity.channel;")
+                .addDirective("exports com.laamella.base.entity.channel.internal to com.laamella.core;")
+                .addDirective("uses com.laamella.base.util.internal.FactoryDelegate;");
+
+        moduleDeclaration.getModuleStmts()
+                .addLast(new ModuleExportsStmt()
+                        .setName("foo.bar")
+                        .addModuleName("other.foo")
+                        .addModuleName("other.bar")
+                );
+
+        moduleDeclaration
+                .addDirective(new ModuleExportsStmt()
+                        .setName("foo.bar.x")
+                        .addModuleName("other.foo")
+                        .addModuleName("other.bar")
+                );
+
+        assertEqualsNoEol("@SuppressWarnings(\"module\") \n" +
```

Comment:
With my annotation prettifying PR in, you need to remove the space before the newline.